openrtb_ext/imp_adhese.go

```diff
@@ -0,0 +1,56 @@
+package openrtb_ext
+
+import (
+	"encoding/json"
+
+	"github.com/mxmCherry/openrtb"
+)
+
+type ExtImpAdhese struct {
+	Account  string          `json:"account"`
+	Location string          `json:"location"`
+	Format   string          `json:"format"`
+	Keywords json.RawMessage `json:"targets,omitempty"`
+}
+
+type AdheseOriginData struct {
```

Comment:
Lines 16 on better fit inside `adapters/adhese/adhese.go` itself or even on a `adapters/adhese/utils.go` file in order to restrict the use of these types to the Adhese adapter only.